src/sparsezoo/utils/onnx.py

```diff
@@ -50,6 +51,30 @@
 ]
 
 
+def onnx_includes_external_data(model: ModelProto) -> bool:
+    """
+    Check whether the ModelProto in memory includes the external
+    data or not.
+
+    If the model.onnx does not contain the external data, then the
+    initializers of the model are pointing to the external data file
+    (they are not empty)
+
+    :param model: the ModelProto to check
+    :return True if the model was loaded with external data, False otherwise.
+    """
+
+    initializers = model.graph.initializer
+
+    data_saved_to_disk = any(
+        [initializer.external_data for initializer in initializers]
```

Comment:
```suggestion
        initializer.external_data for initializer in initializers
```
nit: let's take advantage of generators here?